src/test/java/uk/gov/pay/ledger/util/fixture/QueueRefundEventFixture.java

```diff
@@ -0,0 +1,186 @@
+package uk.gov.pay.ledger.util.fixture;
+
+import au.com.dius.pact.consumer.dsl.PactDslJsonBody;
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.model.SendMessageResult;
+import com.google.common.collect.ImmutableMap;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonParser;
+import com.google.gson.JsonPrimitive;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.apache.commons.lang3.StringUtils;
+import uk.gov.pay.ledger.event.model.Event;
+import uk.gov.pay.ledger.event.model.ResourceType;
+import uk.gov.pay.ledger.rule.SqsTestDocker;
+
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+
+public class QueueRefundEventFixture implements QueueFixture<QueueRefundEventFixture, Event> {
```

Comment:
This class shares quite a lot with (there are two more comments talking about it) https://github.com/alphagov/pay-ledger/blob/8b2385ef8502053d95c8b79457babe07fa80d900/src/test/java/uk/gov/pay/ledger/util/fixture/QueueEventFixture.java
It would be nice to have shared code extracted.

Reply:
Code has been extracted to Util class https://github.com/alphagov/pay-ledger/compare/8b2385ef8502053d95c8b79457babe07fa80d900..e41dffb0f31eb635371f3439403fa7a401018618#diff-27426f41fab61e239f1f824a66a05cb2R13